pallets/pool-system/src/tests/ratios.rs

```diff
@@ -0,0 +1,163 @@
+use sp_arithmetic::traits::EnsureSub;
+
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+use super::*;
+
+#[test]
+fn ensure_ratios_are_distributed_correctly() {
+	new_test_ext().execute_with(|| {
+		let pool_owner = DEFAULT_POOL_OWNER;
+		let pool_owner_origin = RuntimeOrigin::signed(pool_owner);
+
+		// Initialize pool with initial investments
+		let senior_interest_rate = Rate::saturating_from_rational(10, 100)
+			/ Rate::saturating_from_integer(SECONDS_PER_YEAR)
+			+ One::one();
+
+		assert_ok!(PoolSystem::create(
```

Comment:
Q: Would the default pool from `crate::util::default_pool::create()` suffice or is any of the diverging configurations necessary?
* 1.1% annual senior rate vs 0%
* 10% min risk buffer  vs 0%
* 10k AUSD reserve vs 0

Reply:
well, `max_reserve` is a problem. The rest would be fine.

Reply:
I adapted the default pool and used that one.